src/OneExplorer/OneExplorer.ts

```diff
@@ -463,7 +463,10 @@ export class OneNode extends vscode.TreeItem {
     //
     // However, resourceExtname returns info of vscode Explorer view (not of OneExplorer).
     //    "when": "view == OneExplorerView && viewItem == config"
-    this.contextValue = node.typeAsString;
+    //
+    // To resolve above the issue, it appends the extname to the contextValue.
+    const extname = path.extname(node.uri.fsPath);
+    this.contextValue = node.typeAsString + extname;
```

Comment:
For what use would you need extname in contextValue? ❓

Reply:
This code requires to distinguish the tvn file from product files.
I will use the extname like this.
```
        {
          "command": "one.explorer.inferModel",
          "when": "view == OneExplorerView && viewItem =~ /tvn/",
          "group": "inline"
        },
```
from #1854

Reply:
Aha, understood :-D